frontend/src/lib/search.ts

```diff
@@ -46,10 +46,10 @@ export function queryStatusCount(cluster: string): IRequestResult<ISearchResult>
                 },
                 {
                     filters: [
-                        { property: 'compliant', values: ['!Compliant'] },
+                        { property: 'compliant', values: ['NonCompliant'] },
```

Comment:
I am concerned about whether this is equivalent. I think the previous query may have included policies that are not yet reporting status?

Is the `!` operator no longer supported? There are a number of other places where it is used. For example:
https://github.com/stolostron/console/blob/main/frontend/src/routes/Infrastructure/Clusters/ManagedClusters/components/StatusSummaryCount.tsx#L129

Search for `'!` to see others.